### DISCUSSION TITLE
安装后没有新手指引

### DISCUSSION THREAD

**Discussion by denloh** [windingwind/zotero-better-notes#454] [Q&A] [answered]
安装插件成功后重启没有新手指引，Help菜单栏中也没有该选项。Mac版本Zotero 6.0.26

**Comment by denloh** [comment]
尝试新建主笔记没有找到选项
<img width="268" alt="image" src="https://github.com/windingwind/zotero-better-notes/assets/25156418/c73084be-fe3f-4723-bff4-cd40c714f8ce">

**Comment by GeMSeven** [comment]
我也出现这个问题，windows 版本zotero 6.0.26。

**Comment by windingwind** [comment, marked as answer]
最新版本移除了例如初次安装指引等内容。使用文档请参阅主页
